Assets/Group1/Scripts/Enemy.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+[RequireComponent(typeof(BoxCollider2D))]
+public class Enemy : MonoBehaviour
+{
+    public UnityAction<Enemy> DeadEnemy;
```

Comment:
event

Reply:
https://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods